rotkehlchen/externalapis/beaconchain/service.py

```diff
@@ -213,7 +219,10 @@ def _query_chunked_endpoint_with_pagination(
         The offset unfortunately also starts from latest entry so no way to store
         anything to avoid extra calls at the moment.
         """
-        chunks = calculate_query_chunks(indices_or_pubkeys)
+        chunks = calculate_query_chunks(
+            indices_or_pubkeys=indices_or_pubkeys,
+            chunk_size=80,
```

Comment:
small comment why 80
```suggestion
            chunk_size=80,  # small comment
```